src/scripts/main.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
+  const promise1 = (resolve, reject) => {
+    element.addEventListener(eventName, () => {
+      let result = `It was ${eventName} on the element: `;
+
+      result += `${element.nodeName}, id: ${element.id}.`;
+      resolve(result);
+    });
+  };
+
+  return new Promise(promise1);
 }
 
 const printMessage = (message) => {
-// write your code here
+  const div = document.getElementById('div');
```

Comment:
here you should create div element by JS, add css class and append message to it, code can be:

```
const divTag = document.createElement('div');

  divTag.classList.add('message');

  const text = document.createTextNode(`${message}`);

  divTag.appendChild(text);

  document.body.appendChild(divTag);
```